### PR TITLE
refactor: `value-comma_group` node

### PR DESCRIPTION
**Pull request by evilebottnawi**
Second refactoring step

### CODE REVIEW

File: src/language-css/printer-postcss.js [hunk 1]
```diff
@@ -461,166 +474,142 @@ function genericPrint(path, options, print) {
...
-          isNextForKeyword ||
-          isNextEachKeyword
-        ) {
-          parts.push(line);
+
+          continue;
         }
+
+        // Be default all values go through `line`
+        parts.push(line);
```
Comment (evilebottnawi) [inline src/language-css/printer-postcss.js]: Just add own logic before this for formatting and it is all, easy read = easy maintain

**Comment by evilebottnawi** [comment]
After merge solve https://github.com/prettier/prettier/issues/4492

File: src/language-css/printer-postcss.js [hunk 2]
```diff
@@ -39,19 +39,30 @@ const isSCSSControlDirectiveNode = utils.isSCSSControlDirectiveNode;
...
 const isDetachedRulesetCallNode = utils.isDetachedRulesetCallNode;
 const isPostcssSimpleVarNode = utils.isPostcssSimpleVarNode;
 const isSCSSMapItemNode = utils.isSCSSMapItemNode;
+const isInlineValueCommentNode = utils.isInlineValueCommentNode;
+const isHashNode = utils.isHashNode;
+const isLeftCurlyBraceNode = utils.isLeftCurlyBraceNode;
+const isRightCurlyBraceNode = utils.isRightCurlyBraceNode;
+const isWordNode = utils.isWordNode;
+const isColonNode = utils.isColonNode;
+const isMediaAndSupportsKeywords = utils.isMediaAndSupportsKeywords;
```
Comment (j-f1) [inline src/language-css/printer-postcss.js]: What do you think about (maybe in a future PR) getting rid of all of these declarations and just prefixing the variables with `utils.` when they’re used?

Comment (evilebottnawi) [reply src/language-css/printer-postcss.js]: @j-f1 maybe, i think we can import `utils` as `u` for less breaking lines (example in `if`)

**Pull request merged**